orchestrator/cosmos_gravity/src/build.rs

```diff
@@ -229,9 +229,12 @@ pub fn ethereum_event_messages<CS: CosmosSigner>(
     }
 
     let mut msgs = Vec::new();
+    let mut lastNonce: U256 = U256::zero();
```

Comment:
```suggestion
    let mut last_nonce: U256 = U256::zero();
```
got this warning when try to build
```
warning: variable `lastNonce` should have a snake case name
   --> cosmos_gravity/src/build.rs:232:13
    |
232 |     let mut lastNonce: U256 = U256::zero();
    |             ^^^^^^^^^ help: convert the identifier to snake case: `last_nonce`
```